packages/gatsby-source-contentful/src/__tests__/gatsby-plugin-image.js

```diff
@@ -8,7 +8,7 @@ import _ from "lodash"
 import { resolve } from "path"
 import { setFieldsOnGraphQLNodeType } from "../extend-node-type"
 import { generateImageSource } from "../gatsby-plugin-image"
-import * as gatsbyCoreUtils from "gatsby-core-utils"
+import * as fetchRemoteFileModule from "gatsby-core-utils/fetch-remote-file"
```

Comment:
matching how it's imported in the source module we're testing. this mismatch stopped working in node 20.